x-pack/plugins/cases/public/components/create/form.tsx

```diff
@@ -156,12 +168,7 @@ export const CreateCaseForm: React.FC<CreateCaseFormProps> = React.memo(
     timelineIntegration,
   }) => (
     <CasesTimelineIntegrationProvider timelineIntegration={timelineIntegration}>
-      <FormContext
-        afterCaseCreated={afterCaseCreated}
-        caseType={caseType}
-        hideConnectorServiceNowSir={hideConnectorServiceNowSir}
-        onSuccess={onSuccess}
-      >
+      <FormContext afterCaseCreated={afterCaseCreated} caseType={caseType} onSuccess={onSuccess}>
```

Comment:
FormContext no longer takes in the prop `hideCOnnectorServiceNowSir` I checked on main. and it wasnt being used I should say. so I removed it from here